webapp/www/templates/home.html

```diff
@@ -45,43 +45,43 @@
 <fullpage class="instant-size" page-class="page">
```

Comment:
I am not sure we need the second nav bar - don't you think it looks a bit
weird? I just added it because the full page template required it.

You could just try removing it and see how it looks

Shankari